openslides_backend/action/sort_generic.py

```diff
@@ -142,3 +142,48 @@ def create_write_request_elements(
             yield WriteRequestElement(
                 events=[event], information=information, user_id=self.user_id
             )
+
+
+class LinearSortMixin(BaseAction):
+    """
+    Provides a mixin for linear sorting.
+    """
+
+    model: Model
```

Comment:
Just inherit from `Action`, then you can remove this line

Reply:
Done.